extensions/switch-game-play-history/src/components/DailyPlayHistoriesSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { Color, List } from "@raycast/api";
+import { IRecentPlayHistory } from "../types/nintendo";
+import dayjs from "dayjs";
+
+export const DailyPlayHistoriesSection = ({
+  history,
+  timeBarMaxValue,
+}: {
+  history: IRecentPlayHistory;
+  timeBarMaxValue: number;
+}) => {
+  const date = dayjs(history.playedDate).format("YYYY-MM-DD");
+  const totalPlayedMinutes = history.dailyPlayHistories.reduce((acc, cur) => acc + cur.totalPlayedMinutes, 0);
+  const sectionSubtitle = `${history.dailyPlayHistories.length} Game · ${totalPlayedMinutes} Minutes`;
+  return (
+    <List.Section title={date} subtitle={sectionSubtitle}>
+      {history.dailyPlayHistories.map((item) => {
+        const timeBar = {
+          tag: {
+            value: [
+              "\u200B",
+              "\u3000".repeat(Math.floor((item.totalPlayedMinutes / timeBarMaxValue) * 12)),
+              item.totalPlayedMinutes + " min",
+              "\u200B",
+            ].join(""),
+            color:
+              item.totalPlayedMinutes < 60 ? Color.Green : item.totalPlayedMinutes < 120 ? Color.Orange : Color.Red,
+          },
+        };
```

Comment:
This is not the way to use tags but it's creative 🙂 

You could also use [getProgressIcon](https://developers.raycast.com/utilities/icons/getprogressicon) with a tooltip instead 🙂

Reply:
Do I have to change it? If this way is not allowed, then I can only cancel the display of the progress bar. GetProgressIcon only has the style of Ring, which doesn't quite meet my needs.

Reply:
Yeah, that's unfortunately at the moment not allowed, so it needs to be just a tag without the text length 🙂